p2p/client_test.go

```diff
@@ -20,14 +18,17 @@ type TestLogger struct {
 }
 
 func (t *TestLogger) Debug(msg string, keyvals ...interface{}) {
+	t.t.Helper()
```

Comment:
Just out of curiosity: what does this line do?

Reply:
Helpers are skipped/popped from call stack during various operations in tests.
 - `t.Log` reports file/line of actual caller of helper function (in this case it may be something like `client.go:67: DEBUG: starting P2P client` instead of `client_test.go:26: DEBUG: setting up DHT`)
 - errors are reported in more useful way (again, caller of helper function, not helper function itself).